### ISSUE TITLE
'NoneType' object is not iterable

### ISSUE THREAD

**Issue by pbadenski** [slank/awsgi#33]
Getting error below since upgrading to 0.1.0:

```
[ERROR] TypeError: 'NoneType' object is not iterable
Traceback (most recent call last):
  File "/var/task/lambda.py", line 11, in handler
    return awsgi.response(app, event, context)
  File "/var/task/vendor/awsgi/__init__.py", line 25, in response
    sr = StartResponse(base64_content_types=base64_content_types)
  File "/var/task/vendor/awsgi/__init__.py", line 41, in __init__
    self.base64_content_types = set(base64_content_types) or set()
```

**Comment by amaula** [comment]
Yes, noticed the same.

The "set(base64_content_types) or set()" gets "None" for the set() constructor which is invalid.

**Comment by slank** [comment]
This is fixed in PR #34. Reviews welcome.

**Comment by slank** [comment]
This is a fairly trivial and critical fix. I'm going ahead with the merge
